src/appFunctions.js

```diff
@@ -22,6 +32,30 @@ export function findPossiblyOwners(owners, usersApp) {
     ));
 }
 
+export function findCardParentByIdChild(idChild, cards) {
+    let card = null;
+    
+    Object.values(cards).find(array => (
+        card = array.find(item => item.children.includes(idChild))
+    ));
+
+    return card;
+}
+
+export function findAvalibleChildsToAdd(id, cards) {
+    const generalArrayCards = Object.values(cards).reduce((prev, item) => [...prev, ...item]);
+    
+    return generalArrayCards.filter((itemFilter) => {
+        if ((id === itemFilter._id) || itemFilter.children.includes(id)) {
+            return false;
+        }
+        
+        return !generalArrayCards.find((itemFind) => (
```

Comment:
itemFind is a bad one, you don't need find prefix here, use some simple word like card, item etc